src/main/resources/build-release-candidate.bpmn

```diff
@@ -0,0 +1,141 @@
+      "type": "text",
+      "id": "Field_05xq34i"
+    }
+  ],
+  "type": "default",
+  "id": "Form_1ituz9d"
+}</zeebe:userTaskForm>
+      <zeebe:userTaskForm id="UserTaskForm_2j128s3">{
+  "schemaVersion": 2,
+  "components": [
+    {
+      "label": "Current release process",
+      "type": "textfield",
+      "id": "Field_01aedoc",
+      "key": "release_version"
+    },
+    {
+      "text": "# Release the Maven artifacts\n\nOnce the pipeline was successfully finished, the Docker artifacts will already be available publicly. We now have to release the Maven central ones.\n\n1. Go to [Maven Central](https://s01.oss.sonatype.org/) and login using the shared `Maven Central` credentials for login `camundabpm` found in [Keeper](https://keepersecurity.eu/vault/) under `Shared-Engineering - Zeebe`. \n2. On the left side, click on the 'Staging Repositories' link.\n3. After it has been loaded, scroll down to the bottom of the list. You should find the relevant Zeebe staging repositories there.\n4. Mark the repository you want to release, which should be one for zeebe.\n\n&gt; io.camunda:zeebe*\n\n\n5. Click on 'Release' at the menu on top of the list. A window will pop up were you can enter a description but it is not necessary. Activate 'Drop repository after release automatically'. Then proceed.\n6. Done.",
+      "type": "text",
+      "id": "Field_10avz1w"
+    }
+  ],
+  "type": "default",
+  "id": "Form_1tx0oak"
+}</zeebe:userTaskForm>
+    </bpmn:extensionElements>
+    <bpmn:startEvent id="StartEvent_1" name="Build release candidate">
+      <bpmn:outgoing>Flow_1yahb20</bpmn:outgoing>
+    </bpmn:startEvent>
+    <bpmn:userTask id="Activity_0wjyoge" name="Trigger release candidate">
+      <bpmn:extensionElements>
+        <zeebe:formDefinition formKey="camunda-forms:bpmn:UserTaskForm_38jg436" />
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_1yahb20</bpmn:incoming>
+      <bpmn:outgoing>Flow_0laljwe</bpmn:outgoing>
+    </bpmn:userTask>
```

Comment:
:wrench: We could adjust the form to make it more specific for building the release candidate. This is the same for the other user tasks in the process.

I guess that it uses the form `release_trigger_release.form`.

We could change the variable `release_version` to `release_version + "-rc1"`.